controls/aiprompt/styling/inputview-styling.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Input View Styling
+page_title: .NET MAUI AIPrompt Documentation - Input View Styling
+description: Learn how to style the editor and the input button inside the input view of the Telerik .NET MAUI AIPrompt control.
+position: 1
+slug: aiprompt-inputview-styling
+---
+
+# Input View Styling
+
+The Input View of the AIPrompt control provides the following customization properties:
+
+* `InputEditorStyle`(`Style`)&mdash;Defines the style that is to be applied to the input editor.
```

Comment:
Defines the style that is to be applied to the input editor. -> Defines the style applied to the input editor.